monitoring/evidently/data-quality-report/data-quality-report-component.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kfp.components import create_component_from_func, InputPath, OutputPath"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASE_IMAGE = \"quay.io/ibm/kubeflow-notebook-image-ppc64le:latest\"\n",
+    "\n",
+    "def data_quality_report(\n",
+    "    cur_path: InputPath('CSV'),\n",
```

Comment:
1. rename: cur_path -> dataset_path
2. It's good to support CSV but we should also support DFs directly. I propose to add another parameter "dataset_type", which is either "DataFrame", "CSV", or "HuggingFace". I'd default to DF. Obviously, the code needs some adaptation to handle that.

Reply:
I added support for dataframe and csv. the dataframe serialization method im using is pickle. We should discuss the best practices for passing dataframes between components. I see that some examples use `np.savez()` and some use pickle or parquet.